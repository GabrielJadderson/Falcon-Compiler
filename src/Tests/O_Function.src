# Code generation, simple function
func adder(i: int, j: int): int
   return i + j;
end adder

write adder(11, 34);
write adder(0-43, 50);
write adder(0, 0);
write adder(1023, 765);

