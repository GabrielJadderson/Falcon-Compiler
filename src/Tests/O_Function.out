45
7
0
1788
